### PR TITLE
Gh 149 service account upgrade

### PR DESCRIPTION
**Pull request by rocky341**
Hi @t92549 have upgraded the helm chart to include permissions to create/view inside the helm chart itself. Currently after an install you would have to run additional permission commands in order to allow the application to function.

### CODE REVIEW

File: gaas-ui/package.json [hunk 1]
```diff
@@ -14,81 +14,89 @@
...
-    "axios-mock-adapter": "^1.19.0",
+    "@types/supertest": "^2.0.11",
+    "@types/webpack": "^4.41.31",
+    "@typescript-eslint/eslint-plugin": "^4.33.0",
+    "@typescript-eslint/parser": "^4.33.0",
+    "accessibility-checker": "^3.1.11",
+    "axios-mock-adapter": "^1.20.0",
+    "babel-core": "^7.0.0-bridge.0",
     "babel-eslint": "^10.1.0",
+    "babel-jest": "^27.3.1",
```
Comment (t92549) [inline gaas-ui/package.json:78]: This is causing dependency issues for me locally

File: gaas-ui/src/index.tsx [hunk 2]
```diff
@@ -1,27 +1,29 @@
-import React from "react";
-import ReactDOM from "react-dom";
-import App from "./components/App";
-import * as serviceWorker from "./serviceWorker";
-import { BrowserRouter } from "react-router-dom";
-import { CssBaseline } from "@material-ui/core";
-import { StylesProvider } from "@material-ui/styles";
-import GlobalStyles from "../src/components/GlobalStyles";
-import "./styles/_App.scss";
+import React from "react"
```
Comment (t92549) [inline gaas-ui/src/index.tsx:1]: Is there a reason all these semi-colons were taken away?

**Review by t92549 — CHANGES_REQUESTED** [review]
This fails locally for me due to lots of instances of `catch (e)`, for example:
```ts
    } catch (e) {
        this.setState({ errorMessage: `Failed to get all namespaces. ${e.toString()}` });
    }
```
results in `Argument of type 'unknown' is not assignable to parameter of type 'AxiosError<any>'.  TS2345`.
Changing these to `catch (e: any)` made it run but I am not sure it is the correct fix.

**Comment by rocky341** [comment]
> This fails locally for me due to lots of instances of `catch (e)`, for example:
> 
> ```ts
>     } catch (e) {
>         this.setState({ errorMessage: `Failed to get all namespaces. ${e.toString()}` });
>     }
> ```
> 
> results in `Argument of type 'unknown' is not assignable to parameter of type 'AxiosError<any>'. TS2345`. Changing these to `catch (e: any)` made it run but I am not sure it is the correct fix.

The best way of doing this seems to be to cast e as Error which I have done. Should be fixed now but let me know if you are happy with this solution or not.

File: gaas-ui/src/index.tsx (hunk 2 above)
Comment (rocky341) [reply gaas-ui/src/index.tsx]: Not sure how that happened. They have been added back in sorry.

File: gaas-ui/package.json (hunk 1 above)
Comment (rocky341) [reply gaas-ui/package.json]: Wasn't actually needed so has been removed.

**Comment by rocky341** [comment]
Hi @t92549 
PR should be ready now for another review.

File: gaas-ui/package.json [hunk 3]
```diff
@@ -14,81 +14,88 @@
...
+    "eslint-plugin-flowtype": "^5.10.0",
+    "eslint-plugin-import": "^2.25.2",
     "eslint-plugin-jsx-a11y": "^6.4.1",
     "eslint-plugin-notice": "^0.9.10",
-    "eslint-plugin-react": "^7.21.5",
+    "eslint-plugin-react": "^7.26.1",
     "eslint-plugin-react-hooks": "^4.1.2",
     "fetch-mock": "^9.11.0",
     "morgan": "^1.10.0",
     "node-sass": "^4.14.1",
```
Comment (t92549) [inline gaas-ui/package.json:93]: I am getting weird issues unless this is set as `^6.0.1` on Ubuntu

Comment (t92549) [reply gaas-ui/package.json:93]: @rocky341 is this repeatable or just my environment being weird?

Comment (t92549) [reply gaas-ui/package.json:93]: When I say "this" I mean `node-sass`

Comment (rocky341) [reply gaas-ui/package.json:93]: @t92549 what kind of issues? Seems to be running fine on my machine. I updated node-sass locally to 6.0.1 and that also seems to run fine for me. I'm happy to change node-sass to 6.0.1 then if you would prefer?

Comment (t92549) [reply gaas-ui/package.json:93]: It's okay, it works fine for node 14 just had issues when using node 16. Also lots of warnings for both but these can be cleaned up later

**Review by t92549 — APPROVED** [review]

**Pull request merged**